.markdownlint-cli2.cjs

```diff
@@ -0,0 +1,13 @@
+const options = require('@github/markdownlint-github').init({
+  "line-length": false,
+  "ol-prefix": false,
+  "no-trailing-punctuation": false,
+  "no-inline-html": false,
+  "first-line-heading": false,
+  // Rules that we want to enable
+  "no-duplicate-heading": false
```

Comment:
I think we'll want to eventually enable this `no-duplicate-heading` rule. 

With Primer docs, anchor links are generated from the headings. When headings are duplicated on a page, the generated anchor links will always go to the first instance of the heading. 

One example is [#autocorrectable heading](https://primer.style/view-components/linting#autocorrectable) which appears twice on different sections of the page, and share the same anchor link.